server/chat-plugins/info.js

```diff
@@ -2519,6 +2524,42 @@ const pages = {
 		}
 		return buf;
 	},
+	globalpunishments(query, user) {
```

Comment:
You're duplicating a lot of code between these two methods - it seems like both could share a lot of the display logic? Addressing this in a followup would be fine as well if you wanted to merge this as is.